site/en/docs/extensions/mv3/migrating_to_service_workers/index.md

```diff
@@ -217,19 +236,18 @@ function buildCanvas(width, height) {
 }
 ```
 
-In the above block we're constructing a canvas element and painting the entire canvas turquoise. To
-migrate to offscreen canvas, replace `document.createElement('canvas')` with `new
-OffscreenCanvas(width, height)`.
+In the above block we're constructing a canvas element. To migrate to offscreen canvas, replace
+`document.createElement('canvas')` with `new OffscreenCanvas(width, height)`.
 
 ```js
-//// background.js
+// service-worker.js
```

Comment:
Why not background.js? It's a background service worker, right? (And not a background page...)

Reply:
Yes, that's right. Look, I did this in a couple of places to try to indicate through _filename_ that this is new. I think the examples don't stand out clearly as new vs old otherwise, because the filename is the same. At a glance it's not super clear which bit of code I should be copying.

I am going to revert this for now but just something to think about, maybe we can change the background color for "old" code or sth. I know that in other places we use `{% Columns %}` but we're not doing a _direct_ comparison in this part of text.